app/scripts/src/content/search.js

```diff
@@ -59,20 +59,24 @@ Search.prototype = {
   findEpisode: function(options) {
     this.findItem({
       success: function(response) {
-        Request.send({
-          method: 'GET',
-          url: this.getEpisodeUrl(response['show']['ids']['slug']),
-          success: function(resp) {
-            if (this.item.episode) {
-              options.success.call(this, JSON.parse(resp));
-            } else {
-              this.findEpisodeByTitle(resp, options);
+        if (!response) {
```

Comment:
Correct me if I understood wrong, but this code block is not needed, right? Looks like what you did in `content-controller.js` is enough.